docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_6.rst

```diff
@@ -44,28 +38,28 @@ Release notes: Open GEE 5.3.6
       .. rubric:: Resolved Issues
 
       .. list-table::
-         :widths: 10 30 55
+         :widths: 35 65
          :header-rows: 1
 
          * - Number
            - Description
-           - Resolution
-         * - 9999
-           - Add description
-           - Add resolution
+         * - 1851
+           - Open GEE does not function properly on systems with a restrictive umask
+         * - 1806
+           - Confusing "geserver.service changed on disk" message after uninstalling Fusion or Server
+         * - 1800
+           - Upgrade from 5.3.4 to 5.3.5 leaves a non-working ExampleSearch
+         * - 1785
+           - Imagery Projects - Transparent areas appears as black

```

Comment:
Minor: just noticed that there's a grammar problem here. Should be `Transparent areas appear as black` (no "s" in "appear").